frappe/core/doctype/rq_job/rq_job.py

```diff
@@ -76,14 +79,35 @@ def job(self):
 		return self._job_obj
 
 	@staticmethod
-	def get_list(filters=None, start=0, page_length=20, order_by="creation desc"):
-		matched_job_ids = RQJob.get_matching_job_ids(filters=filters)[start : start + page_length]
+	def get_list(filters=None, fields=None, start=0, page_length=20, order_by="creation desc", as_list=False):
+		if order_by and order_by != DefaultOrderBy:
+			*_, sort_key, sort_order = order_by.rsplit(" ", maxsplit=1)
+			sort_key = sort_key.removeprefix("`tabRQ Job`.").strip("`")
+			order_desc = "desc" in sort_order.lower()
+		else:
+			sort_key = "creation"
+			order_desc = True
+
+		matched_job_ids = RQJob.get_matching_job_ids(filters=filters)
 
 		conn = get_redis_conn()
 		jobs = [serialize_job(job) for job in Job.fetch_many(job_ids=matched_job_ids, connection=conn) if job]
 
-		order_desc = "desc" in order_by
-		return sorted(jobs, key=lambda j: j.creation, reverse=order_desc)
+		sort_comparator = partial(rq_comparator, sort_key=sort_key)
+		sorted_jobs = sorted(jobs, key=sort_comparator, reverse=order_desc)[start : start + page_length]
```

Comment:
This was deliberately not implemented. If you have 1000s of jobs this will serialize all of them. This is what used to break `bench doctor` completely (probably still does) 

We can't have diagnostics tools fail to load, that defeats the whole purpose.

Some alternate hack for paging can be based on sorted job names maybe (?)